internal/querycoord/meta.go

```diff
@@ -757,11 +675,15 @@ func (m *MetaReplica) showSegmentInfos(collectionID UniqueID, partitionIDs []Uni
 	if len(partitionIDs) == 0 {
 		return segmentInfos
 	}
+
+	partitionIDMap := make(map[UniqueID]struct{})
```

Comment:
provide a util function to support filter key sets from the map instead of this redundant implementation.

Reply:
fixed